docs/docs/installation.mdx

```diff
@@ -245,7 +246,7 @@ anywhere. If you want to use MITIE, you need to
 tell it where to find this file (in this example it was saved in the
 `data` folder of the project directory).
 
-:::caution
+:::caution Deprecation Warning
 Mitie support is likely to be deprecated in a future release.
```

Comment:
Is this actually true? we keep saying so and then deciding to keep it around

Reply:
I mean you got a very valid point there, but as far as I remember there is no alternative for certain languages (was it Japanese or Cantonese 🤷 ). let's poll the channel.

Reply:
I'll remove it for now, until we have a plan to deprecate it.